main.c

```diff
@@ -171,6 +174,19 @@ main(int argc, char *argv[])
 		}
 	}
 
+	/* User selected 'info' mode. */
+	if (info) {
+		/* Print the encryption parameters used for the file. */
+		rc = scryptdec_file_printparams(infile);
+
+		/* Clean up. */
+		if (infile != stdin)
+			fclose(infile);
+
+		/* Finished! */
+		return (rc);
```

Comment:
Is this what we want here?  If `rc` is nonzero, shouldn't we print one of the error messages and `exit(1)`?

Reply:
Added a commit for REBASEing.